components/siddhi-tryit/org.wso2.carbon.siddhi.tryit.ui/src/main/java/org/wso2/carbon/siddhi/tryit/ui/SiddhiTryItClient.java

```diff
@@ -0,0 +1,206 @@
+					stringBuilder.append(gson.toJson(events));
+				}
+			});
+		}
+
+		Pattern eventPattern = Pattern.compile("(\\S+)=\\[(.*)\\]");
+		Pattern delayPattern = Pattern.compile("(delay\\()(\\d)+");
+		Matcher eventPatternMatcher, delayPatternMatcher;
+		int eventStreamAttributeListSize;
+		String[] eventStreamAttributeArray;
+		String inputStreamName;
+
+		//Send event stream
+		try {
+			String[] inputStreamEventArray = eventStream.split("\\r?\\n");
+
+			for (int i = 0; i < inputStreamEventArray.length; i++) {
+				eventPatternMatcher = eventPattern.matcher(inputStreamEventArray[i].replaceAll("\\s", ""));
+				delayPatternMatcher = delayPattern.matcher(inputStreamEventArray[i]);
+
+				if (eventPatternMatcher.find()) {
+					inputStreamName = eventPatternMatcher.group(1);
+					InputHandler inputHandler =
+							executionPlanRuntime.getInputHandler(inputStreamName);
+
+					eventStreamAttributeArray = eventPatternMatcher.group(2).split(",");
+					eventStreamAttributeListSize = eventStreamAttributeArray.length;
+
+					Object object[] = new Object[eventStreamAttributeListSize];
+					for (int j = 0; j < eventStreamAttributeListSize; j++) {
+						Attribute.Type attributeType =
+								executionPlanRuntime.getStreamDefinitionMap().get(inputStreamName)
+								                    .getAttributeList().get(j).getType();
+						switch (attributeType) {
+							case STRING:
+								object[j] = eventStreamAttributeArray[j];
+								break;
+							case INT:
+								object[j] = Integer.parseInt(eventStreamAttributeArray[j]);
+								break;
+							case LONG:
+								object[j] = Long.parseLong(eventStreamAttributeArray[j]);
+								break;
+							case FLOAT:
+								object[j] = Float.parseFloat(eventStreamAttributeArray[j]);
+								break;
+							case DOUBLE:
+								object[j] = Double.parseDouble(eventStreamAttributeArray[j]);
+								break;
+							case BOOL:
+								object[j] = Boolean.parseBoolean(eventStreamAttributeArray[j]);
+								break;
+							case OBJECT:
+								object[j] = eventStreamAttributeArray[j];
+								break;
+						}
+					}
+					if (i == 0) {
+						executionPlanRuntime.start();
+						inputHandler.send(beginSetTime, object);
+					} else {
+						inputHandler.send((beginSetTime +
+						                   (System.currentTimeMillis() - beginSystemTime)), object);
+					}
+				} else if (delayPatternMatcher.find()) {
+					Thread.sleep(Long.parseLong(delayPatternMatcher.group(2)));
+				} else {
+					if (!inputStreamEventArray[i].equals("")) {
+						executionPlanRuntime.shutdown();
+						errMsg = "You have an error in your event stream \"  " +
+						         inputStreamEventArray[i] +
+						         "\n\"." +
+						         " Expected format: &lt;eventStreamName&gt;=[&lt;attribute1&gt;,&lt;attribute2&gt;]";
+						throw new Exception(errMsg);
+					}
+				}
+			}
+
+			Thread.sleep(500);
+			return map;
+		} catch (Exception e) {
+			errMsg = "Error occurred while processing. " + e.getMessage();
```

Comment:
@tishan89 
Thank you for pointing out. This has been slightly changed with commit: https://github.com/thiliA/carbon-event-processing/commit/630b2b6bc55c0afe6bd765d3dd3a5e0640c88aea which now catches a throwable object. Since there could be runtime exceptions such as execution plan validation exceptions, stream processing exceptions it was decided to catch a throwable object.